fail2ban/tests/files/logs/xrdp

```diff
@@ -0,0 +1,37 @@
+#
+# /var/log/xrdp-sesman.log -- should be about the same on any linux distro
+#
+
+# failJSON: { "time": "2022-04-07T12:11:06", "match": true, "host": "10.171.161.151"}
+[20220407-12:11:06] [INFO ] AUTHFAIL: user=badtypist ip=::ffff:10.171.161.151 time=1649351466
+
+# ip injection: 10.171.161.151 should be matched as the host; 192.168.0.1 is an innocent, injected address
+# failJSON: { "time": "2022-04-07T12:11:24", "match": true, "host": "10.171.161.151", "desc": "specifying ip address as username"}
+[20220407-12:11:24] [INFO ] AUTHFAIL: user=192.168.0.1 ip=::ffff:10.171.161.151 time=1649351484
+
+# ip injection: 10.171.161.151 should be matched as the host; 192.168.0.4 is an innocent, injected address
+# failJSON: { "time": "2022-04-07T12:22:02", "match": true, "host": "10.171.161.151", "desc": "more devious log injection"}
+[20220407-12:22:02] [INFO ] AUTHFAIL: user=loginjector ip=192.168.0.4 time=123456789\n[20220407-12:16:59] [INFO ] AUTHFAIL: user=endinjection ip=::ffff:10.171.161.151 time=1649352122
```

Comment:
I understand that due to greedy catch-all (`.+`) the regex would match the injected user as "expected".
Just because I see escaped new-line char `\n` I would like to know whether it is just an artificial line or real test attempt.
I mean whether it'd be different possibly (more escapes or vice versa totally unescaped new-line).

Reply:
That was a literal `\n` entered into the username field and not an actual newline character. (I couldn't figure out how to enter a literal newline character -- at least interactively -- but I'm not sure it's impossible.)

Do you have a suggested alternate to `.+` that would be more appropriate? I'd be happy to test it.

Reply:
well I'm unsure it is possible interactively (probably only via command line only or directly with protocol), but ok.

> Do you have a suggested alternate to .+ that would be more appropriate? I'd be happy to test it.

Not really, excepting probably `(?:\S+|.+)` which would also work due to both anchors `^...$`, but could be faster for the case of "proper" user name without the space, because of different branch in PCRE without all the greedy probations.